cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -86,87 +85,97 @@ Kitematic and then download, start, and connect to an instance of CDAP.
 Docker from a Command Line
```

Comment:
This should be first. This is *THE* way to run Docker and really the *only* way you'll usually see documented.